src/lib/holocene/navigation/navigation-item.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  import Icon from '../icon/icon.svelte';
+  import type { IconName } from '../icon/paths';
+
+  export let link: string;
+  export let label: string;
+  export let icon: IconName;
+  export let tooltip = label;
+
+  const attributes = link.startsWith('http')
+    ? {
+        rel: 'noreferrer',
+        target: '_blank',
+      }
+    : {};
```

Comment:
This is going to give us some trouble in Cloud. All the AppRoutes are full urls so this will always open cloud nav items in a new tab

Reply:
Ugh. Good catch. Do you think we should maybe be more specific? Maybe it's not worth it since we _only_ use this for the feedback link anyway. Also: I hate this code anyway.

Reply:
Might just need an external prop to set this. Feedback/support and potentially status + any future nav item item that links externally